certification/internal/policy/container/base_on_ubi.go

```diff
@@ -1,12 +1,18 @@
 package container
 
 import (
+	"net/http"
 	"os"
 	"path/filepath"
 	"strings"
+	"time"
+
+	"github.com/redhat-openshift-ecosystem/openshift-preflight/certification/errors"
```

Comment:
Nit: can we organize the imports?

Reply:
ctrl + option + O didn't do anything 🤷  but I removed the spaces..

Reply:
make fmt seems to add them back though